src/detect-engine.c

```diff
@@ -84,6 +84,7 @@
 #include "runmodes.h"
 
 #include "reputation.h"
+#include <stdint.h>
```

Comment:
should not be needed, system includes are in suricata-common.h

Reply:
I still see it in detect-engine.h right ?

Could you squash the commits together ?